code/oot.ld

```diff
@@ -64,8 +64,20 @@ SECTIONS
 		*(.patch_ScrubNutUpgradeOne)
 	}
 
-	.patch_EnKoInitBitMask 0x165760 : {
-		*(.patch_EnKoInitBitMask)
+	.patch_EnKoInitCheckForest_165834 0x165834 : {
+		*(.patch_EnKoInitCheckForest_165834)
+	}
+
+	.patch_EnKoInitCheckForest_165878 0x165878 : {
+		*(.patch_EnKoInitCheckForest_165878)
+	}
+
+	.patch_EnKoInitCheckForest_1658c0 0x1658c0 : {
```

Comment:
Can you make the hex digits be uppercase for consistency here? Same in the one below